### DISCUSSION TITLE
How do I pass CLI or API options to prettier when running ALEFix?

### DISCUSSION THREAD

**Discussion by TheButterflyMon** [w0rp/ale#4136] [Q&A - Ask for help with problems] [answered]
I have set up Prettier as the ALE fixer for HTML files as follows:
```
  let g:ale_fixers = {
        \ 'html': ['prettier'],
        \ }
```
In Prettier, I would like to set the "Bracket Line" option as `true` so that Prettier puts the > of a multi-line HTML element at the end of the last line, instead of being alone on the next line. First, I first tried using configuration files as suggested on the Prettier site. I added the following option to my `.prettierrc` file:
```
  "prettier": {
    "bracketSameLine": true
  }
```
This did not work, so I followed the second advice on [https://prettier.io/docs/en/vim.html](https://prettier.io/docs/en/vim.html) and added the option to my .vimrc:
```
let g:ale_html_prettier_options = '--bracket-same-line'
```
This failed, too. How can I provide options to the Prettier fixer so that it works with ALEFix?

**Comment by hsanson** [comment, marked as answer]
The prettier fixer uses this configuration `g:ale_javascript_prettier_options`.  This is used by the fixer for all languages it supports (e.g. javascript, html, etc.)

**Reply by TheButterflyMon** [reply]
Thank you for the answer. It works for all options, except `-bracket-same-line`. For example, `let g:ale_javascript_prettier_options = '--print-width 120'` sets the line length to 120. I suspect that there is a bug somewhere with the  `-bracket-same-line` option.
